File: server/src/main/java/com/my/model/ReleaseFile.java
package com.my.model;

import java.io.Serializable;

public class ReleaseFile implements Serializable {

    private static final long serialVersionUID = -2568613177795333322L;

    private String name;
    private String src;
    private String dest;
    private String type;
    
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public String getSrc() {
		return src;
	}
	public void setSrc(String src) {
		this.src = src;
	}
	public String getDest() {
		return dest;
	}
	public void setDest(String dest) {
		this.dest = dest;
	}
	public String getType() {
		return type;
	}
	public void setType(String type) {
		this.type = type;
	}
    

}

File: client/Updater/Model/ReleaseFile.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Updater.Model
{
    class ReleaseFile
    {
        public String name { get; set; }
        public String src { get; set; }
        public String dest { get; set; }
        public String type { get; set; }

        public String toString()
        {
            return "name:" + name + " src:" + src + " dest:" + dest + " type:" + type;
        }
    }
}
